src/app/faq-block/faq-block.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="faqblock">
+  <h2>{{headline}}</h2>
+  <div class="questionsAnswers">
```

Comment:
Couldn't find this class anywhere - please remove it if it doesn't exist.

Reply:
Done